src/server/db/models/VerificationLogModel.ts

```diff
@@ -14,7 +14,6 @@ const VerificationLogSchema = new Schema<IVerificationLog>({
   email: {
     type: mongoose.Schema.Types.String,
     required: true,
-    unique: true,
```

Comment:
`email` should not be unique. A password reset log and an email verification log for the same email should be able to exist simultaneously.